core/src/main/java/io/aiven/klaw/controller/AnalyticsController.java

```diff
@@ -25,24 +25,29 @@
 @Slf4j
 public class AnalyticsController {
 
+  public static final int DEFAULT_NUMBER_OF_DAYS = 30;
   @Autowired AnalyticsControllerService chartsProcessor;
 
   @RequestMapping(
       value = "/getTeamsOverview",
       method = RequestMethod.GET,
       produces = {MediaType.APPLICATION_JSON_VALUE})
   public ResponseEntity<List<TeamOverview>> getTeamsOverview() {
-    return new ResponseEntity<>(chartsProcessor.getTeamsOverview(null), HttpStatus.OK);
+    return new ResponseEntity<>(
+        chartsProcessor.getTeamsOverview(null, DEFAULT_NUMBER_OF_DAYS), HttpStatus.OK);
   }
 
   @RequestMapping(
       value = "/getActivityLogForTeamOverview",
       method = RequestMethod.GET,
       produces = {MediaType.APPLICATION_JSON_VALUE})
   public ResponseEntity<TeamOverview> getActivityLogForTeamOverview(
-      @RequestParam("activityLogForTeam") String activityLogForTeam) {
+      @RequestParam("activityLogForTeam") String activityLogForTeam,
+      @RequestParam(value = "numberOfDays", required = false, defaultValue = "30")
```

Comment:
```suggestion
      @RequestParam(value = "numberOfDays", required = false, defaultValue = "" + DEFAULT_NUMBER_OF_DAYS)
```

this suggestion is not so great, but just trying to reuse the constant

Reply:
I'm just going to check if their is an aesthically nicer way to write that.